source/Reloaded.Mod.Launcher/Pages/Dialogs/EditModPages/Complete.xaml

```diff
@@ -1,71 +1,106 @@
-﻿<default:ReloadedPage x:Class="Reloaded.Mod.Launcher.Pages.Dialogs.EditModPages.Complete"
-              xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-              xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
-              xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
-              xmlns:local="clr-namespace:Reloaded.Mod.Launcher.Pages.Dialogs.EditModPages"
-              xmlns:default="clr-namespace:Reloaded.WPF.Theme.Default;assembly=Reloaded.WPF.Theme.Default"
-              xmlns:componentModel="clr-namespace:System.ComponentModel;assembly=WindowsBase"
-              mc:Ignorable="d" 
-              Width="Auto"
-              Height="Auto"
-              Title="Special">
+﻿<default:ReloadedPage
```

Comment:
Running re-format on files makes things very hard to edit, it's best to:

1. Specify what was used to reformat (for consistency across files)
2. Reformat in separate PR, if desired.